docs/api/platform/PowerManagement.md

```diff
@@ -81,6 +81,16 @@ UNLOCK: mbed_rtx_idle.cpp, ln: 131, lock count: 1
 
 <span class="notes">**Note:** Sleep tracing is a debug feature and should only be enabled during development cycle. Its heavy use of UART can affect the device performance.</span>
 
+There is a small trade-off between power saving and memory footprint. When tickless mode is enabled, the memory footprint of your application is slightly bigger. Therefore, if tickless mode does not fit your application requirement, you can disable tickless and achieve further memory optimization. To do that, override the default configuration in the application `mbed_app.json`:
```

Comment:
Would this content fit better in our power optimization section (https://os.mbed.com/docs/mbed-os/v5.14/tutorials/power-optimization.html)?

Reply:
🤷... @bulislaw ?

Reply:
No, the power management page should be referenced in the tutorial. What we are focusing on here is "this low power feature which is on by default may not always work for you".